node_package/src/reactHydrate.ts

```diff
@@ -1,10 +1,13 @@
-import ReactDOM from 'react-dom';
 import { ReactElement, Component } from 'react';
 import supportsReactCreateRoot from './supportsReactCreateRoot';
 
+const reactDomClient = "react-dom/client";
+const reactDom = "react-dom";
+// eslint-disable-next-line import/no-dynamic-require
+const ReactDOM = require(supportsReactCreateRoot ? reactDomClient : reactDom);
```

Comment:
Try `const ReactDOM = require(`react-dom${supportsReactCreateRoot ? '/client' : ''}`)`

Might solve resolution error l

@summera @justin808

Reply:
@tomdracz I can certainly give this a try if you think it'll resolve the load error. I've tried a bunch of combinations without much luck. Do you mind explaining why this would solve the issue?